lua/kitty-scrollback/windows.lua

```diff
@@ -71,8 +71,15 @@ end
 M.open_paste_window = function(start_insert)
   vim.cmd.stopinsert()
 
+  if vim.env.KITTY_SCROLLBACK_NVIM_EDIT_INPUT then
```

Comment:
consider adding a mode env var that can be used in the future for other special cases

Reply:
will do this in the future if there is ever a case for this